tests/api/test_users.py

```diff
@@ -0,0 +1,129 @@
+import random
+
+import pytest
+from faker import Faker
+from tests.api.conftest import fake
+from tests.api.constants import URL
+import requests
+
+
+class TestUserPostRequests:
+    fake = Faker()
+
+    def test_create_user_with_list(self):
+        required_payload = [
```

Comment:
Please move duplicated objects to a separate file. e.g payload_generator.py

NOTE: Once the object is assigned to a variable, it would return the same value. 
Here you need to create a function and call it in tests to generate payload every time